README.md

```diff
@@ -70,10 +70,10 @@ to modify the build scripts. For example, to disable the local build cache when
 
 ### Develocity settings
 
-| Develocity API                        | System property                        | Environment variable                   |
-|:--------------------------------------|:---------------------------------------|:---------------------------------------|
-| gradleEnterprise.server               | gradle.enterprise.url                  | GRADLE_ENTERPRISE_URL                  |
-| gradleEnterprise.allowUntrustedServer | gradle.enterprise.allowUntrustedServer | GRADLE_ENTERPRISE_ALLOWUNTRUSTEDSERVER |
+| Develocity API                  | System property                 | Environment variable            |
+|:--------------------------------|:--------------------------------|:--------------------------------|
+| develocity.server               | develocity.url                  | DEVELOCITY_URL                  |
```

Comment:
I mentioned below but we'll need to keep the ability to set these with the old properties for compatibility with the BVS and CI injection.

Reply:
CI auto-injection does not use these properties to instrument Gradle builds. Instead, an init script directly configures the Develocity Gradle Plugin.

CI auto-injection does require backwards compatibility of many such properties for auto-injection of Maven builds. However, for Maven, these are handled by the Develocity Maven Extension, so they will not be in scope of any changes to Common Custom User Data Maven Extension.

I still agree with keeping backwards compatibility with the old properties for users.

Reply:
The build validation scripts are the same as the CI plugins. They configure the Develocity Gradle plugin via an init script.

Reply:
I now fall back to the GE props in https://github.com/gradle/common-custom-user-data-gradle-plugin/pull/273/commits/fc08a7d1f36eae6bdc1c24378c3c25321a00d37a. But I think it is better to not document those and encourage people to use new properties instead.